doc/jsk_topic_tools/scripts/not_node.rst

```diff
@@ -0,0 +1,32 @@
+not_node.py
+===========
+
+
+What is this?
+-------------
+
+A node that takes a bool value and returns the result of the ``not`` operation.
+
+
+Subscribing Topics
+------------------
+
+* ``~input`` (``std_msgs/Bool``)
+
+  input bool value.
+
+
+Publishing Topics
+-----------------
+
+* ``~output`` (``std_msgs/Bool``)
+
+  The result of the ``not`` operation.
+
+
+Parameters
+----------
+
+* ``~rate`` (Int, Default: ``100``)
+
+  Publishing rate [Hz].
```

Comment:
Could you add usege of the node with topic remapping?

```
rosrun jsk_topic_tools xxx input01:=yy
```

Or, I think creating launch is also useful. (both is OK)

```
roslaunch jsk_topic_tools xxx.launch input01:=yyy 
```

Reply:
Thanks. I added a sample.